server/controllers/api.py

```diff
@@ -811,6 +824,7 @@ def get(self, user, email=None):
 
 # Other
 api.add_resource(Enrollment, '/v3/enrollment/<string:email>/')
+api.add_resource(CourseEnrollment, '/v3/course_enrollment/<string:cid>')
```

Comment:
Also I think the URL route should be a bit more modular: 
I'd suggest the route: `/v3/course/<offering:offering>/enrollment` 

I think `/v3/course/cal/cs61a/fa16/enrollment`  is better `/v3/course_enrollment/4`